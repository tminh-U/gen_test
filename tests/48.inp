9
2 20 14 10 14 6 15 1 14 
10
6 6
9 9
1 5
7 9
4 4
5 9
4 7
6 6
9 9
9 9
