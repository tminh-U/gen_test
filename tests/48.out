2
2
14
5
2
11
10
2
2
2
